src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -114,38 +121,57 @@ const PIACollectionUseAndDisclosure = () => {
             </a>
             {Messages.CollectionNotice.DrafterInput.Title.PartThree.en}
           </label>
-
-          <div className="section__question-hint">
-            {Messages.CollectionNotice.DrafterInput.Description.en}
-          </div>
+          {!isReadOnly && (
+            <div className="section__question-hint">
+              {Messages.CollectionNotice.DrafterInput.Description.en}
+            </div>
+          )}
           <div className="richText" id="drafterDisclosure">
-            <MDEditor
-              id="collectionNoticeDrafter"
-              preview="edit"
-              value={
-                collectionUseAndDisclosureForm?.collectionNotice?.drafterInput
-              }
-              defaultTabEnable={true}
-              onChange={(value) => {
-                stateChangeHandler(value, 'collectionNotice.drafterInput');
-              }}
-            />
+            {isReadOnly ? (
+              <MDEditor.Markdown
+                source={
+                  collectionUseAndDisclosureForm?.collectionNotice?.drafterInput
+                }
+              />
```

Comment:
This is debatable. How do we want to show the markdown content. I learned Bing is displaying inside the MDEditor - just as preview, while you're rendering it on the page completely. Let's get some clarity on this

Reply:
The designs do not show the toolbar, just text output.